src/pages/javascript/es6/some-function/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Some Function
+---
+
+## The Some Function
+
+The `some()` function is used for verifying if at least one element of an array meets a given condition. The function returns `true` if the condition is met by one element, and false if any of the elements met the condition
+
+The original syntax of the map function is:
+```javascript
+  arr.some(function callback(currentValue, index, array) {

```

Comment:
Would you mind fixing the indentation here?

Reply:
Done.